applications/MeshingApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -50,9 +50,10 @@ namespace py = pybind11;
 void AddCustomUtilitiesToPython(pybind11::module& m)
 {
     // MeshingUtilities
+    m.def("EnsureProperties", &MeshingUtilities::EnsureProperties);
```

Comment:
you could also expose the utils as a submodule, I recently figured it out and think that it is a good solution: https://github.com/KratosMultiphysics/Kratos/commit/ff553718c1b9d5c981e141219ac93033ecff44ce#diff-3d608579b83d29266b38af995ee7c6d4

Reply:
OK, I will check